ruby/lib/trailer_vote/media_types/release_push_campaign.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_relative 'base_text'
+require_relative 'types/http_url'
+
+module TrailerVote
+  module MediaTypes
+    ##
+    # Media Types for Release Push Campaign
+    #
+    # The release push campaign is used to show a local push notification upon product release.
+    #
+    class ReleasePushCampaign < BaseText
+      media_type 'release_push_campaign', defaults: { suffix: :json, version: 1 }
+
+      validations do
+        version 1 do
+          attribute :release_push_campaign do
+            attribute :message, String
+
+            link :action
+            link :self
```

Comment:
Marco wanted to be able to configure a block of time where the notification should not go off. (while people are sleeping)

Could you add a field for that?